packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/util.ts

```diff
@@ -151,6 +151,7 @@ export async function generateSpokenMathDetails(mathString: string) {
             case "specialCharacter":
                 // We don't want the backslash from special character
                 // to show up in the generated aria label.
+                // eslint-disable-next-line @typescript-eslint/no-unused-expressions
                 piece.content.length > 1
                     ? (convertedSpeech += piece.content.slice(1))
                     : (convertedSpeech += piece.content);
```

Comment:
You can configure `@typescript-eslint/no-unused-expressions` to allow ternaries if you want, see https://eslint.org/docs/latest/rules/no-unused-expressions#options.

Reply:
I find assignment within a ternary to be pretty difficult to maintain/understand. I re-worked this slightly.

@nishasy for visibility as you wrote this. What do you think of transforming what you wrote to:

```typescript 
convertedSpeech +=
                    piece.content.length > 1
                        ? piece.content.slice(1)
                        : piece.content;
```

Reply:
@jeremywiebe Good point! I like it 🚀